packages/core/src/order/order-params.ts

```diff
@@ -1,8 +0,0 @@
-export enum OrderIncludes {
```

Comment:
Hey @bc-rmalyavc can I ask why this file is removed?

Reply:
This file was not used anywhere anymore. Previously it was used in the order-request-sender itself, so that categories could be added via params. Since now categories are included by default, these `OrderIncludes` are not needed anymore